sdcm/scylla_bench_thread.py

```diff
@@ -106,6 +147,11 @@ def __init__(self, stress_cmd, loader_set, timeout, node_list=None, round_robin=
         self.results_futures = []
         self.shell_marker = generate_random_string(20)
         self.max_workers = 0
+        # Find stress mode:
+        #    "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=100"
+        #    "scylla-bench -workload=uniform -mode=read -replication-factor=3 -partition-count=100"
+        self.sb_mode = re.search(r"-mode=(.+?) ", stress_cmd).group(1)
```

Comment:
@dkropachev i check our scylla-bench commands, each has the -mode and -workload. So ideally the pattern will always found. otherwise scylla-bench will failed to start. I don't think it is need to set default value.
Could you suggest to provide how it is better to handle? log warning, raise an exception? or check on config stage?

Reply:
Add additional check to sct_config. So -mode and -workload alwasy be in command and search will not failed

Reply:
can you make sb_mode be of `ScyllaBenchModes` type? then all the if statement could compare without `.value` extraction. This would add more sense to this `ScyllaBenchModes` enum.

Reply:
@soyacz done. Thanks